corehq/apps/casegroups/dbaccessors.py

```diff
@@ -18,5 +18,15 @@ def get_case_group_meta_in_domain(domain):
     return [(group.get_id, group.name) for group in get_case_groups_in_domain(domain)]
 
 
+def search_case_groups_in_domain(domain, search_string, limit=10):
+    count = 0
+    for result in get_case_group_meta_in_domain(domain):
+        if search_string.lower() in result[1].lower():
```

Comment:
there should be very few of these right? I see it's cached for an hour, so it's probably not a big concern

Reply:
yes, there aren't many of these